Initial attempt at scala automated comments for two-fer (#1148)
exercism/website-copy: automated-comments/scala/general/hard_coded_test_cases.md

This solution has hard-coded some of the test data.

> Generally, solutions should not rely on the specific data being tested. Instead, the solutions should solve the problem generally, for any input.

Reviewer: ```suggestion
Generally, solutions should not rely on the specific data being tested. Instead, the solutions should solve the problem, for any input.
```

> Please try again without hard-coding the test data into your solution.

Reviewer 2: ```suggestion
Remove the hard-coded test data from the solution.
```
